test/release/examples/benchmarks/hpcc/hpl.chpl

```diff
@@ -231,8 +231,8 @@ proc replicateD1(Ab, BD) {
 proc replicateD2(Ab, AD) {
   const replAD = {1..n, 1..blkSize}
     dmapped DimensionalDist2D(targetLocales,
-                              new unmanaged BlockCyclicDim(gridRows, lowIdx=1, blkSize),
-                              new unmanaged ReplicatedDim(gridCols));
+                              new owned BlockCyclicDim(gridRows, lowIdx=1, blkSize),
+                              new owned ReplicatedDim(gridCols));
```

Comment:
AFAIK DimensionalDist2D thinks it's using unmanaged for the different dimensions. It ought to use `owned` on them, I'm just surprised this will work without changing something in DimensionalDist2D. Can you double check that it's freeing the DimensionalDist2D instance that frees them, and not just that this function goes out of scope? E.g. write a test that uses this pattern and returns the DimensionalDist2D (or an array / domain over it).

Reply:
Looks like what I had was broken -- switched to pulling the Dim declarations out